types/dmx/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+          "es6"
+      ],
+      "esModuleInterop": true,
```

Comment:
Is this allowed, in order to `import DMX, { Animation } from 'dmx';` in the tests.

Reply:
use `require` to import
```
import DMX = require('dmx');
```

Reply:
Just curious, what about the dmx library tells you to use require instead? The import syntax was working for me, but I'm guessing if you think it's not correct, it 'working' is coincidental/sketchy.

Reply:
The package is UMD library based on documentation. You should follow documentation and source code.
https://github.com/node-dmx/dmx#library-api
https://github.com/node-dmx/dmx/blob/master/index.js#L64

Reply:
> Is this allowed,

No, because it means that projects that don't include `esModuleInterop: true` can't use this package properly.